setup.py

```diff
@@ -183,7 +187,7 @@
         "release": [
             "gitpython==1.0.0",
             "gsutil",
-            "wheel",
+            "wheel==0.24.0",
```

Comment:
This is the version at the time of writing and the version used by the tests, which seems to be the reason for most / many of our dependency pinned versions.